.travis/install_salt.sh

```diff
@@ -44,12 +44,12 @@ install_salt() {
         printf "\n"
         # Unlink allows switching versions,
         # I wish Homebrew had an atomic operation for pinned upgrades
-        if brew list | grep 'saltstack' >/dev/null; then
+        if brew list | grep 'salt' >/dev/null; then
             brew unlink saltstack
         fi
-        brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/9e3a66b6b7ca978bfea86897dcc3391c37f9f0ef/Formula/saltstack.rb
+        brew install --force https://raw.githubusercontent.com/Homebrew/homebrew-core/master/Formula/salt.rb
```

Comment:
Please pin to a specific version, otherwise we'll end with up macOS builders all having different versions depending on what's in master when they come up.

Reply:
oh oops I _was_ doing this but I probably copied the wrong URL at the end